sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -21,9 +21,18 @@ import { ConfigModule } from "@sims/utilities/config";
 import { DatabaseModule } from "@sims/sims-db";
 import { NotificationsModule } from "@sims/services/notifications";
 import { QueueModule } from "@sims/services/queue";
+import { HttpModule } from "@nestjs/axios";
 
 @Module({
   imports: [
+    {
+      ...HttpModule.register({
+        // `timeout` specifies the number of milliseconds before the request times out.
+        // If the request takes longer than `timeout`, the request will be aborted.
+        timeout: 5000,
```

Comment:
I believe that this value should be at least 30 seconds. Is there a best practice around it? Either way, 5 seconds does not seem too much.

Reply:
mm, Actually,  when I was looking for an ideal timeout, I got mixed answers, and most of the articles that I looked were saying 5s, that's the reason I added 5sec. I can change it.

Reply:
To be clear, did you find 5s as a recommendation to be followed or just as an example for those articles?
If there was a recommendation to be 5s, can you please share one article?

Reply:
no @andrewsignori-aot I didn't find any recommendation, it was more like examples, I have updated it to 30s